### PR TITLE
Minor issue/163/special phase file

### PR DESCRIPTION
**Pull request by mihofer**
Special phase file is now tfs formated,
fixing Issue #162, README updated to reflect the current state

### CODE REVIEW

**Comment by mihofer** [comment]
@lmalina, @JoschD @fsoubelet @awegsche , etc. feel free to correct/extend README

File: README.md [hunk 1]
```diff
@@ -20,7 +20,10 @@ Important ones are: `numpy`, `pandas` and `scipy`.
 
 ### Installing
 
-This package is not deployed, hence you need to use the standard git-commands to get a local copy.
+To use the codes, first a local copy should be obtained via git clone. 
+To run the codes, the package has to be either installed using
+`pip install -e /path/to/omc3` or can be
```
Comment (lmalina) [inline README.md]: Maybe write option from github?

File: README.md [hunk 2]
```diff
@@ -29,36 +32,51 @@ This is the new repository ([old one](https://github.com/pylhc/Beta-Beat.src)) o
...
+
 ### Changelog
 
 - Still building up to first release.
 
 ### Implemented
 
 - Main functions: hole_in_one, harpy
-- Utils: logging, iotools, file handlers, entrypoint
+- Utils: logging, iotools, file handlers
```
Comment (lmalina) [inline README.md]: I wouldn't mention utils...

File: omc3/optics_measurements/phase.py [hunk 3]
```diff
@@ -178,16 +190,23 @@ def write_special(meas_input, phase_advances, plane_tune, plane):
...
+                                                            bpm_err * 360,
+                                                            model_value,
+                                                            _to_deg(model_value),
+                                                            minmu1,
+                                                            minmu2,
+                                                            bpm_phase_advance,
+                                                            elems_to_bpms,
+        ])), ignore_index=True)
+
+    tfs.write(join(meas_input.outputdir, f"special_phase_{plane.lower()}.tfs"), special_phase_df)
```
Comment (lmalina) [inline omc3/optics_measurements/phase.py]: "special_phase_" could be a constant, ".tfs" is already a constant in optics_measurements/constants.py

**Pull request merged**